packages/browserify/test/generateConfig.js

```diff
@@ -11,26 +11,28 @@ test('generateConfig - empty policy', async (t) => {
   const scenario = createScenarioFromScaffold({
     defineEntry: () => {},
     defaultConfig: false,
+    expectedResult: { resources: {} }
   })
   const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, { resources: {} }, 'policy matches expected')
+  scenario.checkResult(t, policy, scenario)
```

Comment:
dont use checkResult to assert an unrelated value
just keep the original assertion